projects/angular/src/popover/signpost/signpost.ts

```diff
@@ -50,7 +50,7 @@ export class ClrSignpost {
   constructor(public commonStrings: ClrCommonStringsService) {}
 
   /**********
-   * @property signPostTrigger
+   * @property ClrSignpostTrigger
```

Comment:
The property actually is. `signPostTrigger` so that change is not needed

Reply:
resolved in commit fef091f